--- /home/test/luajitsh_old	2019-04-30 07:14:45.886489547 +0000
+++ ci/build_container/build_recipes/luajit.sh	2019-04-30 08:26:06.408909618 +0000
@@ -2,16 +2,15 @@
 
 set -e
 
-VERSION=2.1.0-beta3
+#VERSION=2.1.0-beta3
 SHA256=409f7fe570d3c16558e594421c47bdd130238323c9d6fd6c83dedd2aaeb082a8
 if [[ "${OS}" == "Windows_NT" ]]; then
   exit 0
 fi
 
-curl https://github.com/LuaJIT/LuaJIT/archive/v"$VERSION".tar.gz -sLo LuaJIT-"$VERSION".tar.gz \
-  && echo "$SHA256" LuaJIT-"$VERSION".tar.gz | sha256sum --check
-tar xf LuaJIT-"$VERSION".tar.gz
-
+git clone https://github.com/linux-on-ibm-z/LuaJIT.git
+cd LuaJIT
+git checkout v2.1
 # Fixup Makefile with things that cannot be set via env var.
 cat > luajit_make.diff << 'EOF'
 diff --git a/src/Makefile b/src/Makefile
@@ -60,8 +59,8 @@
  ##############################################################################
 EOF
 
-cd LuaJIT-"$VERSION"
-patch -p1 < ../luajit_make.diff
+#cd LuaJIT-"$VERSION"
+#patch -p1 < ../luajit_make.diff
 
 # Default MACOSX_DEPLOYMENT_TARGET is 10.4, which will fail the build at link time on macOS 10.14:
 # ld: library not found for -lgcc_s.10.4
