--- /home/test/WORKSPACE_old	2019-04-30 08:35:56.659367741 +0000
+++ WORKSPACE	2019-04-30 08:43:10.499696252 +0000
@@ -32,17 +32,31 @@
     actual = "//external:ssl",
 )
 
+local_repository(
+    name = "boringssl",
+    path = "/home/test/boringssl",
+)
+
+bind(
+    name = "ssl",
+    actual = "@boringssl//:ssl",
+)
+
 # When updating envoy sha manually please update the sha in istio.deps file also
 #
 # Determine SHA256 `wget https://github.com/envoyproxy/envoy/archive/COMMIT.tar.gz && sha256sum COMMIT.tar.gz`
-ENVOY_SHA = "925810d00b0d3095a8e67fd4e04e0f597ed188bb"
+#
+#ENVOY_SHA = "925810d00b0d3095a8e67fd4e04e0f597ed188bb"
 ENVOY_SHA256 = "26d1f14e881455546cf0e222ec92a8e1e5f65cb2c5761d63c66598b39cd9c47d"
 
-http_archive(
+local_repository(
     name = "envoy",
-    strip_prefix = "envoy-" + ENVOY_SHA,
-    url = "https://github.com/envoyproxy/envoy/archive/" + ENVOY_SHA + ".tar.gz",
-    sha256 = ENVOY_SHA256,
+    path = "/home/test/envoy",
+)
+
+local_repository(
+    name = "rules_foreign_cc",
+    path = "/home/test/rules_foreign_cc",
 )
 
 load("@envoy//bazel:repositories.bzl", "envoy_dependencies")
